README.md

```diff
@@ -95,6 +95,8 @@ Options:
   -a, --append                Turns on append results to file mode rather than replace
   -j, --parallel <number>     Run tests in parallel
   -J, --autoParallel          Run tests in parallel (automatically detect core count)
+  --withTags <tag>            Specify which tags to run from the lookup
+  --excludeTags <tag>         Specify which tags to skip from the lookup
```

Comment:
Maybe:
`Specify which tags to run from the lookup` -> `Only test modules from the lookup that contain a matching tag field`

Also we should allow running against multiple tags, maybe `<tag1> <tag2>`?